### PR TITLE
Update script-lab-react-prod.outlook.xml

### PR DESCRIPTION
**Pull request by nokafor**
Made changes to enable Append On Send and added a high resolution icon (necessary to resubmit to the store)

### CODE REVIEW

File: manifests/script-lab-react-prod.outlook.xml [hunk 1]
```diff
@@ -57,7 +57,7 @@
             </DesktopSettings>
         </Form>
     </FormSettings>
-    <Permissions>ReadWriteMailbox</Permissions>
```
Comment (TCourtneyOwen) [inline manifests/script-lab-react-prod.outlook.xml:60]: Does this particular change adversely impact other scenarios?

Comment (nokafor) [reply manifests/script-lab-react-prod.outlook.xml:60]: Not to my knowledge. We can take some time to verify, but it passed the store's validation, so I believe it works as expected

Comment (TCourtneyOwen) [reply manifests/script-lab-react-prod.outlook.xml:60]: And did you validate the append on send scenario. I think I tried this out a long time ago and it worked with the manifest change but can't remember for sure

Comment (nokafor) [reply manifests/script-lab-react-prod.outlook.xml:60]: Yes we validated the Append on Send scenario. For some reason it wasn't working until we changed this permission.

**Review by TCourtneyOwen — APPROVED** [review]

**Pull request merged**